src/components/editor/View.tsx

```diff
@@ -35,4 +35,7 @@ export class View {
     this.file = file;
     this.type = type;
   }
+  clone(): View {
+    return new View(this.file, this.type);
```

Comment:
`clone` generally means making `copy` of an object. But here, the new view is using the reference of the file from previous view ?